internal/security/bootstrapper/config/types.go

```diff
@@ -0,0 +1,80 @@
+/*******************************************************************************
+ * Copyright 2021 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ *
+ *******************************************************************************/
+
+package config
+
+// BootStrapperInfo defines the first stage gate info
+// It is the first stage gate of security bootstrapping
+type BootStrapperInfo struct {
+	Host      string
+	StartPort int
+}
+
+// ReadyInfo defines the ready stage gate info
+// It is the last stage gate of security bootstrapping for
+// Kong, and all other Edgex core services
+type ReadyInfo struct {
+	ToRunPort int
+}
+
+// TokensInfo defines the tokens ready stage gate info
+// for the secretstore (aka. vault-worker)
+type TokensInfo struct {
+	ReadyPort int
+}
+
+// VaultWorkerInfo defines the fields related to
```

Comment:
Use of the term `VaultWork` is only in the compose file. IMO, we should be using SecretStoreSetup. What if changed to use a different Secret Store instead of Vault?

Reply:
i can change that to `SecretStoreWorker` if that makes you feel better

Reply:
Yes, please. Also see my comments on your dev scripts PR that go with this.

Reply:
changed